### ISSUE TITLE
proposal: is.allNot/is.none interface

### ISSUE THREAD

**Issue by bbusschots-mu** [arasatasaygin/is.js#295]
`is.any` and `is.all` are great, but I often find myself needing to make  sure that an array does not contain any of something:

```
is.allNot.string(false, 42, new Date()); // true
is.allNot.string(false, 42, 'today'); // false
```

If it would be accepted I'd be happy to the code and issue a pull request.

**Comment by ironmaniiith** [comment]
That's exact negation of `is.any`. You can always use `!is.any(...)` for this purpose

**Comment by bbusschots-mu** [comment]
@ironmaniiith yes, but one of the big advantages if is.js is that it allows clear more human-friendly syntax.

You can say `!is.string()`, but the library right pushes you to the much clearer `is.not.string()`. Having to write `!is.any()` is a failure of that model that could be very easily fixed by adding a human-friendly interface.

**Comment by ironmaniiith** [comment]
True. I'm not saying we should not have this functionality. Just told how it can be accomplished for now.

**Comment by ouroborus** [comment]
If is.js is going to have `not` (as it does), it should work with `any` and `all`. The syntax should be either `is.not.any`, `is.any.not`, or both, but definitely not `is.anyNot`.
